ui/app/src/components/RecentlyPublishedDashlet/RecentlyPublishedDashletUI.tsx

```diff
@@ -149,43 +127,42 @@ export default function RecentlyPublishedDashletUI(props: RecentlyPublishedWidge
                             </GlobalAppGridCell>
                             <GlobalAppGridCell className="width20">
                               {itemsLookup[path].stateMap.submittedToLive ? (
-                                <FormattedMessage id="words.live" defaultMessage="Live" />
+                                <FormattedMessage id="environment.live" defaultMessage="live" />
                               ) : (
-                                <FormattedMessage id="words.staging" defaultMessage="Staging" />
+                                <FormattedMessage id="environment.staging" defaultMessage="staging" />
```

Comment:
Rollback. Capitalised is fine.
```suggestion
                                <FormattedMessage id="words.staging" defaultMessage="Staging" />
```